src/components/organisms/TimingEventModal/TimingEventModal.tsx

```diff
@@ -158,7 +158,7 @@ export const TimingEventModal: React.FC<TimingEventModalProps> = ({
               </div>
 
               <div className="input-group">
-                <label htmlFor="name">Name your event</label>
+                <label htmlFor="name">Name your experience</label>
```

Comment:
Maybe `EVENT_TAXON` should be added on https://github.com/sparkletown/sparkle/blob/staging/src/settings/taxonomy.ts and used here and in other places.

It is currently KISS that may allow us quick search through and update later on if/when we add proper externalization of strings.